.pylintrc

```
@@ -52,7 +52,7 @@ ignore=CVS
 # ignore-list. The regex matches against paths and can be in Posix or Windows
 # format. Because '\\' represents the directory delimiter on Windows systems,
 # it can't be used as an escape character.
-ignore-paths=
+ignore-paths= blog/migrations
```

If were'd adding spaces, let's be consistent:
```suggestion
ignore-paths =
  blog/migrations/,
```